ImperatorToCK3/Source/CK3Outputter/outLocalization.cpp

```diff
@@ -0,0 +1,44 @@
+#include "outLocalization.h"
+#include <filesystem>
+#include <fstream>
+
+void CK3::outputLocalization(const std::string& outputName, const World& CK3World)
+{
+	std::ofstream english("output/" + outputName + "/localization/replace/converter_l_english.yml");
```

Comment:
nyet, comrade. /localization/english/filename. Watch the languages.
also export your supported languages.yml.